webui/react/src/contexts/Auth.tsx

```diff
@@ -41,10 +42,16 @@ const reducer = (state: State, action: Action): State => {
       return { ...state, checked: true };
     case ActionType.Reset:
       clearAuthCookie();
```

Comment:
**question:** is removal of cookie dependency part of this PR?

Reply:
yep. I think still it would be safer to keep clearing the auth cookie until we're sure non of the endpoints set it.